Content.Shared/_ES/Auditions/RelationshipPrototype.cs

```diff
@@ -0,0 +1,32 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._ES.Auditions;
+
+/// <summary>
+/// This is a prototype for marking relationships
+/// </summary>
+[Prototype("relationship")]
+public sealed partial class RelationshipPrototype : IPrototype
```

Comment:
fine enough but it'd be nice to at least have the framework for more complicated generation like siblings.